### DISCUSSION TITLE
How to resolve schema related error

### DISCUSSION THREAD

**Discussion by raghumys4** [ory/kratos#1655] [Q&A] [answered]
Exception when calling V0alpha1Api#adminCreateIdentity
Status code: 500
Reason: {"error":{"code":500,"status":"Internal Server Error","reason":"Unable to parse validate JSON object against JSON schema.","debug":"no Loader registered for scheme fil","message":"An internal server error occurred, please contact the system administrator"}}
I  am using Using Ory-Java Client API

// Configure HTTP bearer authorization: oryAccessToken
        HttpBearerAuth oryAccessToken = (HttpBearerAuth) defaultClient.getAuthentication("oryAccessToken");
        oryAccessToken.setBearerToken("BEARER TOKEN");
        AdminCreateIdentityBody adminCreateIdentityBody = new AdminCreateIdentityBody();
       AdminCreateIdentityBody body = adminCreateIdentityBody.schemaId("default");
        try {
            Identity result = api.adminCreateIdentity(adminCreateIdentityBody);
            System.out.println(result);
        } catch (ApiException e) {
            System.err.println("Exception when calling V0alpha1Api#adminCre ateIdentity");
        }

**Comment by aeneasr** [comment, marked as answer]
You probably have a typo in your config - `fil://` vs `file://`
